moped-editor/src/views/projects/projectView/ProjectComponents/featureUtils.js

```diff
@@ -0,0 +1,98 @@
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Lots of util files stacking up - I might create a `utils` folder and then have named files inside like `features.js`, `agol.js`, etc. Any input on that would be great. 🙏

Reply:
You have my support! Doubly so with your suggestion that the folder inherits the more generic name like `utils` or `utilities` and the files get more specific names indicating their purpose. 🎯